src/sentry/api/serializers/models/release.py

```diff
@@ -51,7 +52,7 @@ def _user_to_author_cache_key(organization_id, author):
     return f"get_users_for_authors:{organization_id}:{author_hash}"
 
 
-def get_users_for_authors(organization_id, authors, user=None):
+def get_users_for_authors(organization_id, authors, user=None) -> Mapping[str, Mapping[str, Any]]:
```

Comment:
Also realised that since this can return a serialized user, this could be `Mapping[str, UserSerializerResponse | Mapping[str, Any]]`

We could potentially define a `typeddict` for this other format as well. Since it's repeated in a few places, might also be worth defining a type like
```
Author = UserSerializerResponse | <whatever we have for the non-existant users>
```